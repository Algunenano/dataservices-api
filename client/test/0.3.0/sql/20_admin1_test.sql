../../0.2.0/sql/20_admin1_test.sql