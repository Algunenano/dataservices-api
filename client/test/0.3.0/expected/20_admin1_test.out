../../0.2.0/expected/20_admin1_test.out